far/edit.cpp

```diff
@@ -421,10 +421,9 @@ bool Edit::ProcessInsPath(unsigned int Key,int PrevSelStart,int PrevSelEnd)
 			return false;
 		Data.Folder = os::env::expand(Data.Folder);
 	}
-	else
+	else if (!MakePathForUI(Key, Data.Folder))
```

Comment:
Please do not do this. While technically equivalent, the `else if` construct with a condition irrelevant to the condition in the main `if` clause is often confusing and more difficult to interpret.